ui/app/templates/allocations/allocation/task/index.hbs

```diff
@@ -30,8 +30,8 @@
       </div>
     </div>
   {{/if}}
-  <h1 class="title with-flex" data-test-title>
-    <div>
+  <Hds::PageHeader class="job-page-header" as |PH|>
+    <PH.Title data-test-title>
```

Comment:
Further added the Helios PageHeader component here as the right-aligned buttons look nicer with it.